src/components/BackOptionsCard/index.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState } from "react"
+import { PrimaryButton } from "../PrimaryButton"
+import { BottomInfo, Card, HorizontalFlex } from "./styles"
+
+interface BackOptionsCardProps {
+    title: string
+    pledge?: number
+    paragraph: string
+    quantityNumber?: number
+    isReward?: boolean
+}
+
+export const BackOptionsCard = ({ title, pledge, paragraph, quantityNumber, isReward }: BackOptionsCardProps) => {
+
+    const [isChecked, setChecked] = useState(false)
+
+    const handleSelection = () => {
```

Comment:
Por que isso? Que tal isso
```suggestion
    const handleSelectn = () => {
```